### DISCUSSION TITLE
Adding unique composite fields

### DISCUSSION THREAD

**Discussion by irafaelfurtado** [TiagoSilvaPereira/vemto-issues#448] [Ideas & Suggestions]
Hello Thiago, I'm having difficulty adding unique composite fields.
This is very important. If possible, consider adding this feature as it is essential.

**Comment by TiagoSilvaPereira** [comment]
@irafaelfurtado Vemto v2 allows adding this type of index. Thanks for your suggestion. Vemto v2 Pre-Alpha is coming soon

**Reply by irafaelfurtado** [reply]
Hello @TiagoSilvaPereira , thank you for the feedback.
Is there an estimated release date for v2?
It doesn't need to be precise, just a rough estimate.

Thank you!

**Reply by TiagoSilvaPereira** [reply]
Hi @irafaelfurtado I hope to launch a pre-alpha by the end of November
